docs/install.md

```diff
@@ -1,26 +1,76 @@
 # Installing LibreTime
-There two methods of installing LibreTime - [Source](#source) or
-[Ubuntu package](#ubuntu-package).
+There two methods of installing LibreTime - [Source (recommended)](#source) or
+[Ubuntu package (experimental)](#ubuntu-package).
```

Comment:
I'm not sure the Ubuntu package is experimental at this point. It is tested and works without issue. The only issue is that it supports a subset of the distros and versions that LibreTime supports (namely Ubuntu 16.04 and 18.04)

Reply:
The last time I tried to use the Ubuntu package for installing, it failed, so I found an installation video on YouTube that recommended installing from source and it worked really well. I'll change it.